Kitodo/src/main/java/org/kitodo/production/services/data/LdapServerService.java

```diff
@@ -162,6 +162,11 @@ public void createNewUser(User user, String password)
             return;
         }
 
+        if ( user.getLdapGroup().getLdapServer() == null) {
+            Helper.setMessage(Helper.getTranslation("noLdapServerAssignedToLdapGroup"));
```

Comment:
My first intention was, that `Helper.setErrorMessage` should be used here, but it is also a regular message the lines above. 🤦 
Please change to errorMessage in both places, or open an issue and fix it in a separat PR (which would be the better way 😏  )